Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM osrf/ros:humble-desktop-full
+
+# Install dependencies (taken from devcontainer dockerfile)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
```

Comment:
Can you add `wget` to this? This crashes my run of `install_spot_ros2.sh`:
```
./install_spot_ros2.sh: line 35: wget: command not found
```

Reply:
(although I guess this should really be in the `install_spot_ros2` script)

Reply:
I believe I did. Must have forgotten to commit before pushing 😅